xdsl/dialects/csl/csl_wrapper.py

```diff
@@ -303,6 +311,18 @@ def layout_yield_op(self) -> YieldOp:
         """
         return cast(YieldOp, self.layout_module.block.last_op)
 
+    @property
+    def exported_symbols(self) -> Sequence[BlockArgument]:
+        """
+        Get the exported symbols.
+        """
+        return self.program_module.block.args[
+            2 + len(self.params) + len(self.layout_yield_op.fields) :
+        ]
+
+    def set_program_name(self, name: StringAttr):
+        self.program_name = name
```

Comment:
Why would you use this helper instead of assigning to the property directly? Am I missing something here?

Reply:
changing as suggested